doc/data_structures.rst

```diff
@@ -420,6 +420,34 @@ ensemble compatibility features are used.
     but the name provided to ``run_model()`` must be prefixed with
     the ``DataSet`` name in the pattern ``{dataset_name}.tensor_name``.
 
+Support on Systems with Multiple GPUs
```

Comment:
One thing I'm cognizant of is that we have multiple ways of utilizing resources on our DB nodes. One mode that I'm thinking clashes a little bit with the others is the databases per node functionality. I think for now, maybe just add a TODO to check compatibility of your multi-GPU convenience functions with that.

Reply:
Not sure I understand what you're asking for, please elaborate

Reply:
We had (at one point) we had ran multiple shards on the same node. I suppose this comment is partly a question about whether this was tested with that functionality. I'm not sure how these `set/run_multigpu` routines would work in that scenario. I *think* they should be ok, but it could be worth checking at some point if you haven't done it already

Reply:
I don't think that should affect anything, but colocated support was out of scope for this initial work

Reply:
Agreed. I think testing in both co-located and multiple DB per node modes should be tested at some point, but out of scope. Would you mind making a new ticket for that?

Reply:
Done: see SmartSim-Scaling #13